### PR TITLE
Adjust links to other blog articles to be relative

### PR DESCRIPTION
**Pull request by sabine**
Resolves https://github.com/mirage/mirage-www/issues/816.

### CODE REVIEW

File: data/weekly/2015-02-25.md [hunk 1]
```diff
@@ -143,7 +143,7 @@ zone). To this end, Anil has been reinstalling the second Bytemark machine in
 order to run a version of Xenserver. We will also need to sort out
 certificates as we should be running HTTPS for the site.
 
-[mir-www]: http://mirage.io
+[mir-www]:
```
Comment (samoht) [inline data/weekly/2015-02-25.md:146]: seems to be a bit too aggressive :p

File: data/weekly/2015-01-14.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ consistent in using it on our written material.
 
 In terms of practical steps, Amir will update things as he sees them on the
 website and written materials. Anil suggested that we may start using a
-different domain for the site too ([mirage.io](http://mirage.io)).
+different domain for the site too ([mirage.io]()).
```
Comment (samoht) [inline data/weekly/2015-01-14.md:59]: same here, I think we want to keep that link (just fix the https)

File: data/blog/ocaml-tls-api-internals-attacks-mitigation.md [hunk 3]
```diff
@@ -584,7 +584,7 @@ Mirleft, Morocco.
 <img src="/graphics/aftas-mirleft.jpg" alt="Aftas Beach" />
 
 [horizon]: http://www.horizon.ac.uk
-[mirage]: http://mirage.io
+[mirage]:
```
Comment (samoht) [inline data/blog/ocaml-tls-api-internals-attacks-mitigation.md:587]: should be http -> https

Comment (hannesm) [inline data/blog/ocaml-tls-api-internals-attacks-mitigation.md:587]:
```suggestion
[mirage]: https://mirage.io
```

File: data/weekly/2015-01-14.md (hunk 2 above)
Comment (hannesm) [inline data/weekly/2015-01-14.md:59]:
```suggestion
different domain for the site too ([mirage.io](https://mirage.io)).
```

File: data/weekly/2015-02-25.md (hunk 1 above)
Comment (hannesm) [inline data/weekly/2015-02-25.md:146]:
```suggestion
[mir-www]: https://mirage.io
```

**Comment by hannesm** [comment]
I took the liberty to suggest the suggestions from @samoht as corrections and pushed them into this PR. Once CI is green I'll go ahead and merge.

**Comment by hannesm** [comment]
Thanks @sabine

**Pull request merged**